packages/storefront-lwc/src/modules/commerce/lineitemtotal/lineitemtotal.js

```diff
@@ -0,0 +1,38 @@
+/*
+    Copyright (c) 2020, salesforce.com, inc.
+    All rights reserved.
+    SPDX-License-Identifier: BSD-3-Clause
+    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+*/
+import { LightningElement, api } from 'lwc';
+
+export default class LineItemTotal extends LightningElement {
+    @api product;
```

Comment:
Instead of passing in the whole product, maybe we should just pass the props we need for this component.

Reply:
done